src/settings.rs

```diff
@@ -103,14 +103,12 @@ pub struct Settings {
     pub adm_sub1: Option<String>,
     /// adm Endpoint URL
     pub adm_endpoint_url: String,
-    /// max number of tiles returned to non-mobile clients (default: 3)
-    pub adm_max_tiles: u8,
```

Comment:
I am a little surprised that it's easier to make a code change than to set an environment variable to override this, though. Isn't this setting controlled by `CONTILE_ADM_MAX_TILES`? How come we can't set it to `2` to override the new default of 3?